CONTRIBUTING.md

```diff
@@ -0,0 +1,71 @@
+This is a simple guide of the workflow being used in Cryostat. We will use https://github.com/cryostatio/cryostatio.github.io as an example.
+
+## Fork the upstream repository
+
+See https://docs.github.com/en/get-started/quickstart/fork-a-repo
+
+## Local changes
+
+Clone the forked repository.
+```
+$ git clone git@github.com:<username>/cryostatio.github.io.git
+$ git remote add upstream git@github.com:cryostatio/cryostatio.github.io.git # Adding upstream as remote
+```
+Checkout another branch.
+```
+$ cd cryostatio.github.io && git checkout -b some-task-branch
+```
+Edit some files and commits your changes.
+```
+$ git add . # Stage your changes. Here, we stage the entire repo (recursively).
+$ git commit -m "some very nice message"
+```
+
+## Open a PR
+
+Push your changes to your remote respository (forked). You might need to rebase on the newest commit on the upstream.
+
+To rebase:
+```
+$ git fetch upstream # Fetch upstream commits
+$ git checkout main && git merge upstream/main && git push origin main # Checkout main, merge upstream and update origin
+```
+To push:
+```
+git checkout some-task-branch
+git push -u origin some-task-branch
+```
+
+Visit upstream repository and open a PR.
+
+![visit-upstream](https://user-images.githubusercontent.com/68053619/178812137-ee9c1fe3-2f78-4671-8bdc-41c0a55565d7.png)
+
+Compare changes across forks (your branch in fork to the upstream main).
+
+![select-compare-across-forks](https://user-images.githubusercontent.com/68053619/178812907-ea709be4-265a-4f4b-9127-80e4eaf97089.png)
+
+Add: 
+- A title following [guidelines](https://www.conventionalcommits.org/en/v1.0.0/).
+- A description with a referenced issue that the PR is for.
+- A tag (depending on your PR scope).
+- A reference to the current project (i.e. 2.2.0)
```

Comment:
I'd remove this item. This is really the maintainer's concern, not the individual contributor.